lib/livebook/app.ex

```diff
@@ -0,0 +1,120 @@
+if Mix.target() == :app do
+  defmodule Livebook.App do
```

Comment:
If we start adding more functionality around this, I would have this as `LivebookApp` under `lib/livebook_app.ex` (mirroring the CLI). But it can be done in another PR. :)

Reply:
+1

Reply:
Oh I like this, let me make this change right now.